src/styles/banner.scss

```diff
@@ -1,81 +1,53 @@
 button {
-    background: #4ca1af;
-    width: 35%;
-    height: 3rem;
```

Comment:
By using the selector `button` you are styling _any_ button that appears on the page while this stylesheet is present. A safer practice would be for all of these individual stylesheets to start with their container classname and then wrap styles.

example:
```
.banner {
  button: // styles
}
```

Again, don't set widths (or heights) on buttons. They'll break if someone has a different system font size, screen sizes, etc.